### ISSUE TITLE
err: No handler...

### ISSUE THREAD

**Issue by tfer** [mattn/vscode-openvim#2]
I had this installed and working in an earlier version of vscode, but it has trouble installing under 0.10.6, and if I just copy the repo into the extension folder, it still doesn't work.  I get a "No handler found for the command: 'extension.openVim'. Ensure there is an activation event defined, if you are an extension" error.

Perhaps this similar issue has some bearing?; https://github.com/lukasz-wronski/vscode-ftp-sync/issues/1

**Comment by mattn** [comment]
I can't see anything updated on the issue.

**Comment by tfer** [comment]
Working now, does not seem to be any problem with your plugin.

I just manually removed the folder "mattn.OpenVim-0.0.7" at "C:\Users\Tom.vscode\extensions\", and used 'ext install OpenVim' clicking on the download-from-cloud icon.  I got 3 folders: images, node_modules, & out, and 3 files .vsixmanifet, package.json, & readme.md .  For some reason I wasn't getting all this before, perhaps it was something to due with the marketplace problem, anyway, it looks like the partial download was interfering with re-installation attempts.
